code/__DEFINES/atmospherics.dm

```diff
@@ -246,6 +249,13 @@
 
 #define ASSERT_GAS(gas_id, gas_mixture) if (!gas_mixture.gases[gas_id]) { ADD_GAS(gas_id, gas_mixture.gases) };
 
+//prefer this to gas_mixture/total_moles in performance critical areas
+#define TOTAL_MOLES(cached_gases, out_var)\
+	out_var = 0;\

```

Comment:
Didn't we remove this define before because it wasn't worth having around

Reply:
but its getting used here

Reply:
I was misremembering removing some other define for atmos we removed due to maintainability concerns even though it was marginally faster.